piksi_tools/bootload.py

```diff
@@ -78,9 +87,9 @@ def wait_for_handshake(self, timeout=None):
     return True
 
   def reply_handshake(self):
-    # < v2.0 of the bootloader, reuse single handshake message.
-    if self.version < "v2.0":
-      self.link.send(SBP_MSG_BOOTLOADER_HANDSHAKE_DEVICE, '\x00')
+    # < 0.45 of SBP protocol, reuse single handshake message.
+    if self.sbp_version < 45:
```

Comment:
Should sbp_version be compared as a float or version string instead? Integer minor version seems odd.